### PR TITLE
feature(wallet-mobile): new tx review for yoroi swap

### PR DESCRIPTION
**Pull request by banklesss**
##### Ticket

YV-37

NOTE: I refactored the settings folder structure to adopt the new one we are applying to the new features.

### CODE REVIEW

File: apps/wallet-mobile/src/features/Swap/useCases/ReviewSwap/ReviewSwap.tsx [hunk 1]
```diff
@@ -72,38 +65,38 @@ export const ConfirmTxScreen = () => {
...
+      const poolIcon = <PoolIcon providerId={orderData.selectedPoolCalculation.pool.provider} size={18} />
+      const poolProviderFormatted = capitalize(orderData.selectedPoolCalculation.pool.provider)
+      const poolUrl = getPoolUrlByProvider(orderData.selectedPoolCalculation.pool.provider)
+
+      liquidityPool = (
+        <LiquidityPool liquidityPoolIcon={poolIcon} liquidityPoolName={poolProviderFormatted} poolUrl={poolUrl} />
+      )
+    }
+
+    if (liquidityPool != null) customReceiverTitleChanged(liquidityPool)
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Swap/useCases/ReviewSwap/ReviewSwap.tsx:91]: This one will be improved. I will create a map of addresses and dexes

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts [hunk 2]
```diff
@@ -23,9 +23,13 @@ describe('cip30ExtensionMaker', () => {
   })
 
   it('should support getBalance', async () => {
+    console.log('test-1')
     const cip30 = cip30ExtensionMaker(mocks.wallet, mocks.walletMeta)
+    console.log('test2', cip30)
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]: leftovers of dev

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts [hunk 3]
```diff
@@ -23,9 +23,13 @@ describe('cip30ExtensionMaker', () => {
...
     const result = await cip30.getBalance()
+    console.log('test3')
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]: ⬆️

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts [hunk 4]
```diff
@@ -23,9 +23,13 @@ describe('cip30ExtensionMaker', () => {
...
     expect(result).toBeDefined()
+    console.log('test4')
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]: :arrow_up:

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts (hunk 2 above)
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]:
```suggestion
```

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts (hunk 3 above)
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]:
```suggestion
```

File: apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts (hunk 4 above)
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/cip30/cip30.test.ts]:
```suggestion
```

**Review by stackchain — APPROVED** [review]
lgtm

**Pull request merged**